languages/emf-metamodel/src/main/java/de/jplag/emf/util/EMFUtil.java

```diff
@@ -57,14 +55,13 @@ public static void registerEPackageURIs(Collection<EPackage> ePackages) {
 
     /**
      * Loads a model or metamodel from a absolute file path.
-     * @param file is file to the (meta)model.
-     * @return the content of the loaded (meta)model resource or null if it could not be loaded.
+     * @param file is file path to the (meta)model.
+     * @return the resource of the loaded (meta)model or null if it could not be loaded.
      */
-    public static List<EObject> loadModel(File file) {
+    public static Resource loadModelResource(File file) {
         final ResourceSet resourceSet = new ResourceSetImpl();
         try {
-            final Resource resource = resourceSet.getResource(URI.createFileURI(file.getAbsolutePath()), true);
-            return resource.getContents();
+            return resourceSet.getResource(URI.createFileURI(file.getAbsolutePath()), true);
```

Comment:
```suggestion
            return resourceSet.getResource(URI.createFileURI(file.getAbsolutePath()), false);
```

Load on demand should be false here, as the resource is expected to exist

Reply:
Ah, good catch!

Reply:
It seems the test cases cannot load the resources with this set to false...

Reply:
ok, seems I misread or misunderstood the documentation 😅